packages/block-library/src/paragraph/index.js

```diff
@@ -22,6 +22,11 @@ export const settings = {
 	description: __( 'Start with the building block of all narrative.' ),
 	icon,
 	keywords: [ __( 'text' ) ],
+	example: {
```

Comment:
I applied `Needs Dev Note` label as this is block API related and introduces a completely new field. It should also get properly documented in tutorials and related docs.